src/shacl2code/lang/common.py

```diff
@@ -102,12 +102,16 @@ def get(self, _id):
         template = env.get_template(template.name)
 
         classes = ObjectList(model.classes)
+        concrete_classes = ObjectList(
+            list(c for c in model.classes if not c.is_abstract)
+        )
         enums = ObjectList(model.enums)
 
         render = template.render(
             disclaimer=f"This file was automatically generated by {os.path.basename(sys.argv[0])}. DO NOT MANUALLY MODIFY IT",
             enums=enums,
             classes=classes,
+            concrete_classes=concrete_classes,
```

Comment:
Is it possible to add an `abstract_classes` bit here?

Reply:
Sure